setup.py

```diff
@@ -34,6 +36,11 @@ def exec_file(path_segments):
 except Exception:
     pass
 
+class PostInstallCommand(install):
+    def run(self):
```

Comment:
I dont like this idea because the goal of the install command of the setuptools is other. You can try to install the package. You can try to test the results of to understand  it better:

```
python setup.py develop
python setup.py install --user
```